librz/bin/p/bin_ne.c

```diff
@@ -16,6 +16,7 @@ static bool check_buffer(RzBuffer *b) {
 
 	ut16 idx;
 	if (!rz_buf_read_le16_at(b, 0x3c, &idx)) {
+		return false;
 	}
```

Comment:
@08A please send this fix as a separate PR, we will include it in 0.3.0

Reply:
A new PR was opened.